ditto/utils/webhooks.py

```diff
@@ -9,23 +9,27 @@
 
 class EmbedWebhookLogger:
     _to_log: list[discord.Embed]
+    _files_to_log: list[discord.File]
 
     def __init__(self, webhool_url: str) -> None:
         self.loop = asyncio.get_event_loop()
         self._webhook_url = webhool_url
         self._to_log = []
+        self._files_to_log=[]
 
         self._session = aiohttp.ClientSession()
         self._webhook = discord.Webhook.from_url(self._webhook_url, session=self._session)
 
         # setup loop
         self._loop.start()
 
-    def log(self, embed: discord.Embed) -> None:
+    def log(self, embed: discord.Embed,files: list[discord.File]=[]) -> None:
```

Comment:
There's no actual harm in it here, but defaulting to a mutable type is usually undesirable.